From 3fe0c270cd043025dfb1ab187ca0404c13f62d8b Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Thu, 31 Jan 2013 09:26:13 +0000
Subject: [PATCH 0729/1083] thermal: rcar: remove machine_power_off() from
 rcar_thermal_notify()

Machine/System power-off is run in thermal frame work if
it become critical temperature.
This patch removed pointless machine_power_off()
from thermal_zone_device_ops :: .notify

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Zhang Rui <rui.zhang@intel.com>
(cherry picked from commit e6e053f4e47634c07993cb31893556d24e18b65e)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 93b1feb625a68dcda8f6733889b10f9ad7033eb6)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/thermal/rcar_thermal.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/thermal/rcar_thermal.c b/drivers/thermal/rcar_thermal.c
index e31a82a..7d43744 100644
--- a/drivers/thermal/rcar_thermal.c
+++ b/drivers/thermal/rcar_thermal.c
@@ -261,7 +261,6 @@ static int rcar_thermal_notify(struct thermal_zone_device *zone,
 	case THERMAL_TRIP_CRITICAL:
 		/* FIXME */
 		dev_warn(dev, "Thermal reached to critical temperature\n");
-		machine_power_off();
 		break;
 	default:
 		break;
-- 
1.8.3.2

